src/views/received_sms_list/ReceivedSmsList.js

```diff
@@ -7,47 +7,49 @@ import {
     RECEIVED_SMS_LIST_PATH,
     STATUS_ALL,
 } from './config'
-import { DeleteSelectedButton } from './DeleteSelectedButton'
+import DeleteSelectedButton from '../../delete_selected_button/DeleteSelectedButton'
```

Comment:
I'm all for default exports when only a single module is being exported from a file. But it doesn't seem to fit with the other components, judging from the imports here. I'd change all of them at once instead of introducing it just for a couple of files.

Reply:
:100: I'll convert everything to a default export in a different PR and revert `DeleteSelectedButton` back to a single var export.

Reply:
Thanks, sounds good!